src/js/providers/utils/quality-labels.ts

```diff
@@ -10,7 +10,7 @@ export function generateLabel(level?: ProviderLevel, qualityLabels?: QualityLabe
     if (!level) {
         return '';
     }
-    // flash provider uses bitrate instead of bandwidth
+    // custom providers may use level bitrate rather than bandwidth
     const bandwidth = level.bitrate || level.bandwidth;
     // flash provider, in some cases, will create its own label. Prefer it over creating a new label
```

Comment:
Should we update this comment as well with `custom providers`?

Reply:
something like `prefer provider label` or remove it.